clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/CTInAppNotification.java

```diff
@@ -409,6 +409,15 @@ CTInAppNotification initWithJSON(JSONObject jsonObject, boolean videoSupported)
 
     }
 
+    CTInAppNotification copy() {
+        return new CTInAppNotification().initWithJSON(jsonDescription, videoSupported);
```

Comment:
we need not parse the json again, we can assign from fields of old object. can add todo.

Reply:
Added a TODO